pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -21,14 +26,24 @@ const (
 type VPCNetworkConfigurationSpec struct {
 	// PolicyPath of Tier0 or Tier0 VRF gateway.
 	DefaultGatewayPath string `json:"defaultGatewayPath,omitempty"`
+
+	// VPCConnectivityProfile ID. This profile has configuration related to create VPC transit gateway attachment.
+	VPCConnectivityProfile string `json:"vpcConnectivityProfile,omitempty"`
+
+	// +kubebuilder:validation:Enum=SMALL;MEDIUM;LARGE;XLARGE
+	LbServiceSize string `json:"lbServiceSize,omitempty"`
+
 	// Edge cluster path on which the networking elements will be created.
 	EdgeClusterPath string `json:"edgeClusterPath,omitempty"`
+
 	// NSX-T Project the Namespace associated with.
 	NSXTProject string `json:"nsxtProject,omitempty"`
```

Comment:
NSXTProject -> "NsxProject"

Reply:
The VPCNetworkConfigurationSpec should be like 
type VPCNetworkConfigurationSpec struct {
NsxProject
VpcConnectivityProfile
PrivateIPs
ShortID
LbServiceSize
PodSubnetAccessMode
DefaultSubnetSize
}

Reply:
@wenqiq  probably you missed this one?

Reply:
@dantingl  do we only support for VPC new model or both old and new model?